packages/terra-application-links/src/list/ApplicationList.scss

```diff
@@ -26,8 +24,8 @@
 
     &[aria-current='true'] {
       background-image: var(--terra-application-list-selected-background-image, linear-gradient(-270deg, #fff, #e8e9ea));
-      border-left: var(--terra-application-list-selected-border-left 0.5rem solid #007cc3);
-      padding-left: var(--terra-application-list-padding-left 0.7143rem);
+      border-left: var(--terra-application-list-selected-border-left, 0.5rem solid #007cc3);
+      padding: var(--terra-application-list-selected-padding, 0.8225rem 0.7143rem);
```

Comment:
This will work fine in RTL since both left and right paddings are equal in this default theme. There may be a concern in other themes, if the left and right padding are not equal, not entirely sure how the PostCSS RTL plugin will handle this.